src/ui/public/chrome/directives/app_switcher_link/__tests__/app-switcher-link.js

```diff
@@ -207,5 +208,15 @@ describe('appSwitcherLink directive', () => {
         expect(link.attr('title')).to.be(attrs.appSwitcherLinkTitle);
       });
     });
+
+    describe('appSwitcherLinkClasses attribute', () => {
+      it('is applied to the link', () => {
+        const attrs = {
+          appSwitcherLinkClasses: 'new classes',
+        };
+        const element = create(attrs);
+        expect(element.attr('class')).to.contain(attrs.appSwitcherLinkClasses);
```

Comment:
I think that's a very reasonable point. The reason I like this pattern is that it makes intention crystal-clear; there's no doubt what I want to happen in this test because the variable carries meaning. If I'm comparing against a literal value, then I need to read the entire test (or rely on a comment) to extract that meaning. Whenever I read a test and find it confusing, it's usually because I don't understand the intention behind the test.

I don't think I can objectively say this way is better than your suggestion, though, since there are pros and cons to both.